### PR TITLE
Jdr garnett article types

### PR DESCRIPTION
**Pull request by Joshua-David**
## What does this change?
Style for gt news now done

## What is the value of this and can you measure success?
N/A

## Does this affect other platforms - Amp, Apps, etc?
No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?
N/A

## Screenshots
N/A

## Tested in CODE?
Yes

### CODE REVIEW

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 1]
```diff
@@ -1,69 +1,164 @@
-$quote-tail: 25px; // ********** Article Type **********
+$quote-tail: 25px;
+@mixin multiline($number) {
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: we should probably pull this out to `static/src/stylesheets/_mixins.scss`?

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 2]
```diff
@@ -1,69 +1,164 @@
...
+    content: '';
+    display: block;
+    width: 100%;
+    margin-bottom: $gs-baseline/4;
+    background-image: repeating-linear-gradient(#dcdcdc, #dcdcdc 1px, #ffffff 1px, #ffffff 4px);
+    height: $number * 4px;
+}
+// hack to hide left/right page margins, temp
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: could we just remove them? is there a plan to?

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 3]
```diff
@@ -1,69 +1,164 @@
...
+        .submeta__section-labels & {
+            font-size: 16px;
+            line-height: 22px;
         }
     }
 
-    .submeta__link {
-        font-weight: 200;
+    .submeta__section-labels .submeta__link {
+        font-family: 'Guardian Egyptian Web', 'Guardian Text Egyptian Web', Georgia, serif!important;
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: there's a mixin you can use for fonts, in pasteup

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 4]
```diff
@@ -106,15 +233,92 @@ $quote-tail: 25px; // ********** Article Type **********
...
+
+    .caption--main.caption--img {
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background: rgba(22, 22, 22, .8);
+        color: #ffffff;
+        display: none;
+        padding: .375rem 2.5rem .75rem .625rem;
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: go with pixels here – they'll be converted to rems in build, and easier to reason about!

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 5]
```diff
@@ -106,15 +233,92 @@ $quote-tail: 25px; // ********** Article Type **********
...
+        @include fs-headlineGarnett(4);
+        font-weight: 500;
+        @include mq(tablet) {
+            @include fs-headlineGarnett(6);
+            font-weight: 500;
+        }
+    }
+
+    .tonal__head--tone-feature .content__headline {
+        font-weight: bolder;
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: 700?

File: static/src/stylesheets/module/content-garnett/_types.scss (hunk 2 above)
Comment (Joshua-David) [reply static/src/stylesheets/module/content-garnett/_types.scss]: yes but how can we without removing them from live?

**Review by sndrs — APPROVED** [review]

**Pull request merged**